Makefile

```
@@ -2,6 +2,7 @@
 
 VERSION = $(shell ./scripts/version.sh)
 LATEST_VERSION = $(shell git tag | sort -Vr | head -n 1)
+LARKY_PATH = $(shell find . -name "larky-runner")
```

This is a bit risky since potentially larky-runner binary can be pulled if from an unexpected place. Please consider something like this:
```suggestion

LARKY_PATH = $(shell python -c "import pkg_resources; print(pkg_resources.resource_filename('pylarky', 'larky-runner'))")
```